proto/system_config.proto

```diff
@@ -130,11 +130,11 @@ message DaqConfig {
   // Reporting device result
   DeviceReporting device_reporting = 53;
 
-  // For specifying ip subnets assigned by external DHCP server
-  repeated IpSubnet external_dhcp_subnets = 54;
+  // For specifying ip subnets assigned by external DHCP server / static ips
+  repeated string external_subnets = 54;
```

Comment:
in terms of future-proofing, can you think of any potential information (properties) we'd maybe need to keep along with the subnet?  E.g., DNS server or some other property? I can't specifically think of something -- just asking the question to make sure we're considering things.  In general, having this be an object (not necessarily IpSubnet but maybe something like SubnetSpec) makes the system more flexible in the future since we can then add a property to each entry. If it's encoded as a string, then there's no path forward for adding annotations.

In the past, every time I've done something like "a list of strings" then I end up regretting it because I often then want to add some kind of information to it and then you're SOL.